plugins/inputs/postgresql/postgresql.go

```diff
@@ -42,6 +40,20 @@ var sampleConfig = `
   ##
   address = "host=localhost user=postgres sslmode=disable"
 
+  ## connection configuration.
+
+  ## maxidle - specify the number of idle connections to maintain. see sql.DB for details.
```

Comment:
I think the first line is a sufficient description, I simply included the comment for further reading.

Are you implying we need to document it more? or that sql.DB shouldn't be referenced?